mne/epochs.py

```diff
@@ -1546,6 +1549,34 @@ def equalize_event_counts(self, event_ids, method='mintime', copy=True):
             epochs.drop_bad_epochs()
         # figure out how to equalize
         eq_inds = list()
+
+        # deal with hierarchical tags
+        ids = epochs.event_id
+        if "/" in "".join(ids):
+            event_ids = [[x] if isinstance(x, string_types) else x
+                         for x in event_ids]
+            event_ids = [[k for k in ids if all((tag in k.split("/")
+                         for tag in id_))]
+                         if all(id__ not in ids for id__ in id_)
+                         else id_
+                         for id_ in event_ids]
+            for id in event_ids:
+                if len(set([id_ in ids for id_ in id])) != 1:
+                    err = ("Don't mix hierarchical and regular event_ids"
+                           " like in \'%s\'." % ", ".join(id))
+                    raise KeyError(err)
+
+            # deal with non-orthogonal tags
+            events_ = [set(epochs[x].events[:, 0]) for x in event_ids]
+            doubles = events_[0].intersection(events_[1])
+            if len(doubles):
+                warnings.warn("Warning: the two sets of epochs are "
+                              "overlapping. The %s overlapping epochs will"
+                              " be dropped." % len(doubles))
```

Comment:
my argument for dropping them is that they didn't think they would add anything to any contrast. i didn't think about the fact that there could be unequal contribution. i guess an error makes sense if the hierarchical tags should be orthogonal